contrib/init/sysvinit-debian/docker

```diff
@@ -119,9 +119,13 @@ case "$1" in
 	stop)
 		check_init
 		fail_unless_root
-		log_begin_msg "Stopping $DOCKER_DESC: $BASE"
-		start-stop-daemon --stop --pidfile "$DOCKER_SSD_PIDFILE" --retry 10
-		log_end_msg $?
+		if [ -f "$DOCKER_SSD_PIDFILE" ]; then
+			log_begin_msg "Stopping $DOCKER_DESC: $BASE"
+			start-stop-daemon --stop --pidfile "$DOCKER_SSD_PIDFILE" --retry 10
+			log_end_msg $?
+		else
+			log_warning_msg "Docker already stopped - file $DOCKER_SSD_PIDFILE not found."
```

Comment:
No, because it is not an error to try to stop a service that is not running. I was trying to install docker on VPS. I installed it but I couldn't run it because of the VPS restrictions. So I wanted to uninstall docker, but I couldn't because docker was not running.